### PR TITLE
Feature/api calls

### LINKED ISSUE #10: Create fetch requests

### PR DESCRIPTION
**Pull request by BrandonAinsworth**
What does this PR do?
- Adds functionality to fetch our data from our endpoints

Where should your reviewer start?
- Line 1 in apiCalls.js
- Line 7 in App.js
- Line 1 in Form.css
- Line 1 in Form.js
- Line 1 in Header.js
- Line 3 in Home.js
- Line 6 in index.js

What (if any) features are you implementing?
- GET 

What (if anything) did you refactor?
- N/A

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- We have reseeded our data, as of writing this it has not updated our endpoints.

Resolves #10

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -14,12 +14,16 @@ class App extends Component {
       allZipCodes: [],
       selectedZip: '',
       filteredTrees: [], 
-      selectedTreee: ''
+      selectedTree: ''
```
Comment (irmakerdem) [inline src/components/App/App.js:17]: Nice job catching a spelling error ✍️

File: src/components/Form/Form.js [hunk 2]
```diff
@@ -1,9 +1,42 @@
-import React from "react";
+import React, { Component } from 'react';
```
Comment (irmakerdem) [inline src/components/Form/Form.js:1]: Good job keeping quotation marks consistent! 👏

**Review by Baskanbetul — APPROVED** [review]
It was perfect team working , nice job using import / export.

File: src/components/Form/Form.js [hunk 3]
```diff
@@ -1,9 +1,42 @@
...
+        <form onSubmit={(event) => this.props.changeZipCode(this.state.zip, event)}>
+            <p>Zip Code:</p>
+            <input
+                className='zip-code-entry'
+                type='text'
+                data-cy='zip-code-entry'
+                value={this.state.zip}
+                name='zip'
+                placeholder='Enter Zip Code'
+                pattern='[0-9]{5}'
```
Comment (irmakerdem) [inline src/components/Form/Form.js:29]: WHOA a pattern attribute to make sure zip code is 5 characters long for error handling!! 😎

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -1,9 +1,12 @@
-import React from "react";
+import React from 'react';
 import './Header.css';
+import { Link } from 'react-router-dom';
 
 const Header = () => {
     return(
-        <h1>Header</h1>
+        <Link to='/'>
+            <h1 className='header' data-cy='header'>🌳 Tree Pal 🌳</h1>
```
Comment (irmakerdem) [inline src/components/Header/Header.js:8]: Look how cute the 🌳s emojis are!! 🥰

**Review by irmakerdem — APPROVED** [review]
Components are off to a great start!! 😃

**Review by rachel-bock — APPROVED** [review]
Looks good.  🌴 🌳

**Pull request merged**